server/commands/cycle.js

```diff
@@ -1,47 +1,65 @@
 /* global graphQLFetcher, notifyUser, commandsConfig */
 
-function invokeLaunchCycleAPI(lgJWT) {
+function invokeUpdateCycleStateAPI(state, lgJWT) {
   const baseURL = process.env.NODE_ENV === 'development' ? 'http://game.learnersguild.dev' : 'https://game.learnersguild.org'
   const mutation = {
-    query: 'mutation { launchCycle { id } }'
+    query: 'mutation($state: String!) { updateCycleState(state: $state) { id } }',
+    variables: {state},
   }
   return graphQLFetcher(lgJWT, baseURL)(mutation)
     .then(data => data.launchCycle)
 }
 
-function launchCycle(commandInfo) {
+function handleUpdateCycleStateCommand(commandInfo, state, msg) {
   try {
     const {lgJWT, lgPlayer} = Meteor.user().services.lgSSO
```

Comment:
Moderators are not necessarily players, and players are not necessarily moderators. This destructuring will likely fail because there won't be an `lgPlayer` attribute on the Meteor user record for moderators. The good news is, you shouldn't need the `lgPlayer` info, because only moderators can manipulate cycles. So ... just get rid of it.